wordpress-mobile/WordPress-iOS
# Disable Local Notifications for WeeklyRoundupBackgroundTask + Remove pending notifications in settings

This PR disables adds code to cancel pending notifications when a user modifies WordPress notification settings and also disables local notifications for WeeklyRoundupBackgroundTask

Ref #19560 

To test:
WeeklyRoundup Notifications Test
1. Test on device.  When WPiOS app launches go to Me -> App Settings -> Debug.  Tap Weekly Roundup
2. If you don't have enough blogs / history also slide to the right to "Include A8c P2s"
3. Tap one of the Schedule in 10 sec option or longer and then minimise the app in background
4. Weekly Round up Notification should not appear

UI Notification Settings Test
5. Test on device.  When WPiOS  app launches on main tab bar choose Notifications and then Notification Settings on upper right
6. Click on one of your Sites -> Push Notifications7. 
7.  WeeklyRoundup should no longer be a row (you can compare to production AppStore app where Weekly Roundup is the last row) 

## Regression Notes
1. Potential unintended areas of impact
Weekly Roundup Background Task

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested on app

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
